plugins/com.google.cloud.tools.eclipse.appengine.flex/plugin.xml

```diff
@@ -35,6 +35,8 @@
             class="com.google.cloud.tools.eclipse.util.service.ServiceContextFactory:com.google.cloud.tools.eclipse.appengine.newproject.flex.AppEngineFlexProjectWizard"
             icon="platform:/plugin/com.google.cloud.tools.eclipse.appengine.ui/icons/gae-16x16.png"
             project="true"
+            finalPerspective="org.eclipse.jst.j2ee.J2EEPerspective"
+            preferredPerspectives="org.eclipse.jst.j2ee.J2EEPerspective,org.eclipse.wst.web.ui.webDevPerspective"
```

Comment:
Will adding the Java perspective at the end help users installing our plugin on non-J2EE Eclipse versions?